packages/rollup-plugin-html/test/src/input/extract/extractModules.test.ts

```diff
@@ -164,7 +164,7 @@ describe('extractModules()', () => {
         attributes: [],
       },
     ]);
-    expect(moduleImports).to.eql([]);
```

Comment:
See [packages/rollup-plugin-html/src/input/getInputData.ts](https://github.com/modernweb-dev/web/pull/2571/files#diff-613f8b096f9083d34f31f4948192f2fcf924c508dd6980b7197971c94ade0c61)
The order between inline and other modules was lost because they was gathered in two different lists. Then they was merged together upon return, making the inline modules always come after the other modules. I changed it to put all modules including inline modules in the sam list from the start. Thus, the moduleImports lists is no longer empty if there is inlineModules.